tensorboard/plugins/histogram/histograms_plugin_test.py

```diff
@@ -120,29 +149,32 @@ def test_index(self):
                 'description': self._HTML_DESCRIPTION,
             },
         },
-    }, self.plugin.index_impl())
-
-  def _test_histograms(self, run_name, tag_name, should_work=True):
-    self.set_up_with_runs([self._RUN_WITH_SCALARS,
-                           self._RUN_WITH_LEGACY_HISTOGRAM,
-                           self._RUN_WITH_HISTOGRAM])
+    }, plugin.index_impl(experiment='exp'))
+
+  @with_runs([
```

Comment:
By, the decorator is very cool!

thinking out loud: it seems other tests can benefit from this decorators throughout our codebase. :thinking:

Reply:
Yeah, they can certainly be helpful when used judiciously!